public/components/overview/github-panel/config/main.tsx

```diff
@@ -28,7 +28,7 @@ export const MainViewConfig = {
                 tableTitle='Actors'
                 aggTerm='data.github.actor'
                 aggLabel='Actor'
-                maxRows={5} 
+                maxRows={100} 
```

Comment:
**thought**: I am not sure why this change.

Reply:
I have modified it by a variable, to extract the same data from the 4 fields. Thanks for the warning.